platforms/hyperledger-fabric/configuration/add-new-channel.yaml

```diff
@@ -0,0 +1,113 @@
+# This playbook is a subsequent playbook
+##########################################################################################
+# DO NOT RUN THIS IF YOU HAVE NOT RUN deploy-network.yaml and deployed the Fabric network
+# and want to add a new channel to the existing network
+##########################################################################################
+# This playbook can only be run after all pods and orderers are available
+# Please edit the same network.yaml according to the documenation in docs/sourc/operations/add_new_channel_fabric.md.
+# Refer the example under platforms/hyperledger-fabric/configurations/samples/network-fabric-add-new-channel.yaml
+---
+  # This will apply to ansible_provisioners. /etc/ansible/hosts should be configured with this group
+- hosts: ansible_provisioners
+  gather_facts: no
+  tasks:
+
+    # delete build directory
+    - name: Remove build directory
+      file:
+        path: "./build"
+        state: absent
+
+    # Create or get Orderer crypto materials
+    - include_role:
+        name: "create/crypto/orderer"
```

Comment:
Do we require the crypto role (orderer & peer) for adding a new channel?

Reply:
Yes, channel creation does need the crypto material of both orderers and the peer as far as I understand the steps.